shared/terraform.go

```diff
@@ -38,18 +46,36 @@ func loadTFconfig(
 	t *testing.T,
 	varDir string,
 	terraformOptions *terraform.Options,
-	product string,
 ) (*Cluster, error) {
 	c := &Cluster{}
+	product := os.Getenv("ENV_PRODUCT")
```

Comment:
we should avoid 4 equal calls to os. to get the same data.

 you are calling it here and on  setTerraformOptions

Reply:
is it ok if I update, as it allows to not get env vars on clusterconfig file?
```
cfg, err = config.AddEnv()
	if err != nil {
		shared.LogLevel("error", "error adding env vars: %w\n", err)
		os.Exit(1)
	}
cluster = shared.ClusterConfig(cfg)
```
also this makes sense to pass the config on the cluster, may be rename to envConfig?
or should I do
```
func ClusterConfig() *Cluster {
    product := os.Getenv("ENV_PRODUCT")
	module := os.Getenv("ENV_MODULE")
	once.Do(func() {
		var err error
		cluster, err = newCluster(product, module)
```





2:57
to reduce repetitions…